pkg/build/config/version.go

```diff
@@ -150,7 +151,11 @@ func CheckDroneTargetBranch() (VersionMode, error) {
 	if reRlsBranch.MatchString(target) {
 		return ReleaseBranchMode, nil
 	}
-	return "", fmt.Errorf("unrecognized target branch: %s", target)
+	if rePRCheckBranch.MatchString(target) {
+		return PullRequestMode, nil
+	}
+	fmt.Printf("unrecognized target branch: %s, defaulting to %s", target, PullRequestMode)
+	return PullRequestMode, nil
```

Comment:
This won't do anything other than prevent a warning being printed, right? Don't we want to change the final `return` to something else?

Reply:
I don't think we want to throw an error here, we only want to make sure that the user understands that it's running in pull request mode because the target branch didn't match any pattern.